markdown/introduction.md

```diff
@@ -5,6 +5,18 @@ Nav: home.nav
 
 TableOfContents
 
+## {id="what"} What does it, really?
+
+What does it mean that "Bedrocklinux allows to run a linux system that is composed of different linux distributions"? It means, the system boots with a bootloader, a kernel, and an init system that can be independently selected from the "bedrocked" distributions, and can then freely run userland packages that are installable side-by-side from the different distributions.
+
+Consequently, the running system is rooted in different linux distributions.
+Therefore, a descriptive name for bedrocklinux concept could actually be multi-rooted-linux.
+
+In a pretentious language it could also have been marketed as something like "distrosnip", because, in a way, a multi-rooted system takes to an end what appimages, flatpacks or snap packages had begun.
```

Comment:
Bedrock predates flatpak and snap.

Reply:
Now that is really cool. Didn't know that.
```suggestion
In a pretentious language it could also have been marketed as something like "distrosnip", since in a way, bedrock linux had already thought through and been solving what flatpacks or snap packages only came to struggle with later.
```